tests/score_variants/test_score_function.py

```diff
@@ -53,3 +53,59 @@ def test_eq_score():
     assert score_function.get_score(3) == 3
     assert score_function.get_score(12) == 12
     assert score_function.get_score(None) == not_reported_score
+
+def test_score_mode_user_defined_range():
+    """ Test score mode w user defined override"""
+    score_function = ScoreFunction(match_type='integer', equal=True)
```

Comment:
Use the #GIVEN-#WHEN-#THEN syntax in unit tests. Thin helps explain the test (see cg for an example)